dependency-includer-script.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Dependency includer script.
+ * TODO: use Composer autoloading instead of this.
+ *
+ * @package Newspack
+ */
+
+if ( ! class_exists( '\NewspackContentConverter\Config' ) ) {
```

Comment:
Refresh my memory why you took this approach, as opposed  to simply including all of the `include_once`s in the main plugin file?

Reply:
Great one. Thanks for the suggestion. Updated, removed all the `if`s.